charts/opentelemetry-demo/values.yaml

```diff
@@ -139,6 +139,11 @@ components:
   #   configuration: {}
   # # Kubernetes container health check options
   #   livenessProbe: {}
+  # # Optional init container to run before the pod starts. This is implemented as a netcat check to the port specified.
```

Comment:
Do we need the second sentence about netcat?  That feels specific to the featureflagservice.

Reply:
good catch, forgot to update that line.